### ISSUE TITLE
Cannot enter the value ".5" in number cells

### ISSUE THREAD

**Issue by delavager** [angular-ui/ng-grid#2279]
This is a little weird, but I tested it on the http://ui-grid.info/docs/#/tutorial/201_editable demo and it produces the same result in the "Age" column.

Basically I want to be able to enter the value ".5" without preceding it with a 0.  If I, or a user, types "." nothing happens, no value is entered and no error.  If I start with a 0 and type "0.5" it works fine.  This is really silly but my users are just going to cry foul when they try and type ".5".

My set up is using the row editing feature and has enableCellEditOnFocus: true.
The columns have a cellFilter of 'number : 2' (which shouldn't matter as that only applies to the non-editable template) and are aggregated to a footer for a column total.

I also have a calculated column where each object has a function called "TotalCalc()" which aggregates 5 values across the row.  At first I thought it was this, but removing the aggregations did not fix the problem.

Thanks.

**Comment by PaulL1** [comment]
I think perhaps this could get rolled up into a bootstrap feature, refer #2290

**Comment by PaulL1** [comment]
Closing - this is a browser limitation not something ui-grid can fix.  We have provided the ability to create custom editors with non-html5 widgets, which potentially would provide a way around this.

**Comment by delavager** [comment]
Is it a browser issue if it's happening with 3 different browsers on more than one persons machine?

**Comment by PaulL1** [comment]
Yes.  Some browsers don't support html5 widgets, and some browsers that do support html5 widgets don't allow entry of `.5` when the entry type is number.  It's a fault in the implementation of the widget within that browser (so it would obviously happen on more than one person's machine), and there's nothing we can do about it.

The best answer is to use a widget set other than html5 widgets, but we've made an active choice not to pick just one widget set - we'd prefer to be open to many, hence why we've allowed people to create custom editors.
